### PR TITLE
must have token in event.header

### PR DESCRIPTION
**Pull request by hunshenshi**
https://app.clickup.com/t/861mp97dt

### CODE REVIEW

File: pkg/modules/event/event.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package event
 import (
 	"context"
 	"errors"
+	"github.com/machinefi/w3bstream/pkg/modules/project"
+	"github.com/machinefi/w3bstream/pkg/modules/publisher"
```
Comment (CoderZhi) [inline pkg/modules/event/event.go:7]: wrong group

File: pkg/modules/event/event.go [hunk 2]
```diff
@@ -126,37 +126,31 @@ func checkHeader(ctx context.Context, projectName string, l log.Logger, ret *Han
...
-		if len(r.Header.EventId) > 0 {
-			ret.EventID = r.Header.EventId
-		}
-		if len(r.Header.EventType) > 0 {
-			eventType = r.Header.EventType
-		}
+	var (
+		pub *models.Publisher
+		prj *models.Project
+	)
```
Comment (CoderZhi) [inline pkg/modules/event/event.go:132]: no need to define here

File: pkg/modules/event/event.go [hunk 3]
```diff
@@ -126,37 +126,31 @@ func checkHeader(ctx context.Context, projectName string, l log.Logger, ret *Han
...
+	pub, err = publisherVerification(ctx, l, r)
```
Comment (CoderZhi) [inline pkg/modules/event/event.go:133]: what's the different between line 133 and line 143?

**Comment by saitofun** [comment]
fixed #470

**Review by saitofun — APPROVED** [review]

**Pull request merged**